runtime/src/bank.rs

```diff
@@ -606,10 +609,31 @@ impl Bank {
             .collect()
     }
 
+    /// Return the number of ticks since genesis.
     pub fn tick_height(&self) -> u64 {
         self.last_id_queue.read().unwrap().tick_height
     }
 
+    /// Return the number of ticks since the last slot boundary.
+    pub fn tick_index(&self) -> u64 {
```

Comment:
What’s the plan to remove these constants?

Reply:
The first step is to add them to GenesisBlock and use them to set member variables in Bank. Eventually, they should be updated as we process the ledger and observe votes on cluster parameters.

Want me to add the GenesisBlock part as part of this PR or add it just after?

Reply:
After is fine with me, I just wanted to ensure you weren’t operating under the assumption that those defaults are fixed.

Reply:
I'll update `LeaderScheduler::new_with_bank()` to allow these to comfortably coexist as we transition more to `LeaderSchedule::new_with_bank()` (note the missing R)